core/src/test/resources/recursive_model_1.yaml

```diff
@@ -19,6 +19,7 @@ components:
   schemas:
     B:
       type: object
+      required: ["message2"]
```

Comment:
The test `RecursiveSchemaTest` expects backwards-incompatible change and by coincidence used removal of optional field from response as a change. Either the field should be required or the test should expect backwards-compatible change. I decided to go with the former.